recipes/benchpots/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "benchpots" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 12726b7db046ba0f3ca04a4be8b3a3075608e55fb666a85791d97d91d59a43d6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - h5py
+    - numpy
+    - pandas
+    - pyarrow
+    - scikit-learn
+    - pytorch >=1.10.0
+    - tsdb >=0.4
+    - pygrinder >=0.6
```

Comment:
These are all run dependencies, no? Unless the package imports itself at build time, which you should report upstream b/c it is a bug, you don't need these here not in the test section. You do need them in the run section though.

```suggestion
```